### PR TITLE
Update forms ACG and CCG links

### PR DESCRIPTION
**Pull request by andresriveratoro**
[27930](https://jira.devops.va.gov/browse/API-27930)

### CODE REVIEW

**Review by gileswells — APPROVED** [review]
Simple

File: src/containers/documentation/RequestSandboxAccess.tsx [hunk 1]
```diff
@@ -7,32 +7,18 @@ import { Link } from 'react-router-dom';
 import { PageHeader } from '../../components';
 import { LPB_APPLY_URL } from '../../types/constants';
 import { ApplySuccessResult } from '../../types/forms/apply';
-import {
-  AUTHORIZATION_CCG_PATH,
-  AUTHORIZATION_PKCE_PATH,
```
Comment (Cgfor3) [inline src/containers/documentation/RequestSandboxAccess.tsx:12]: We should remove these constants from our paths file as well

Comment (gileswells) [reply src/containers/documentation/RequestSandboxAccess.tsx:12]: Very soon, we should be able to use these path constants to to track down all instances of these legacy URLs and update them accordingly.

Comment (andresriveratoro) [reply src/containers/documentation/RequestSandboxAccess.tsx:12]: I can remove these in the next PR

Comment (Cgfor3) [reply src/containers/documentation/RequestSandboxAccess.tsx:12]: They only exist for the sandbox and prod access forms. The local sandbox form component isn't used as it's been replaced, and Andres is updating the production form in another PR so there shouldn't be other instances of these URLs

Comment (Cgfor3) [reply src/containers/documentation/RequestSandboxAccess.tsx:12]: JK Routes.tsx line 36 is double redirecting now

`    <Route path="/oauth" render={(): JSX.Element => <Redirect to="/explore/authorization" />} />
`

**Review by Cgfor3 — DISMISSED** [review]
LGTM

**Review by Cgfor3 — APPROVED** [review]
LGTM

**Pull request merged**